.env.sample

```
@@ -1,5 +1,10 @@
-PANO_MINIO_URL=127.0.0.1:9000
-PANO_MINIO_ACCESS_KEY=zlGjx3eZm9JLEc39A87f
-PANO_MINIO_SECRET_KEY=yPhvV67qhUItegOSTdKbhX1eerZSHe86SciwJ6Td
-PANO_MINIO_BUCKET=panoramas
-PANO_MINIO_SECURE=False
+PG_CONN=postgresql://pano-dev:pano-dev@localhost:5432/pano-dev
+
+MINIO_URL=127.0.0.1:9000
+MINIO_ACCESS_KEY=zlGjx3eZm9JLEc39A87f
+MINIO_SECRET_KEY=yPhvV67qhUItegOSTdKbhX1eerZSHe86SciwJ6Td
```

These guys can both be set to `pano-dev`
```suggestion
MINIO_ACCESS_KEY=pano-dev
MINIO_SECRET_KEY=pano-dev
```

---

To confirm, MinIO doesn't expect them to be, say, Base64 strings of a certain byte length?

---

Seems like it. I usually don't configure them in the docker compose and just set them as `sampleaccesskey` and `samplesecretkey`. Give it a try and tell me if I'm crazy.